CHANGELOG.md

```diff
@@ -4,6 +4,18 @@
 All notable changes to the CLDF specification will be documented in this file.
 The CLDF specification adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [1.3] - ...
+
+### Added
+
+- New component `ParameterNetwork` and associated properties.
+- New module `TextCorpus` and associated properties.
+- New property `treeReference` to relate data to objects in `TreeTable`
+- New language property `parentLanguageGlottocode`.
+- New language property `speakerArea`.
```

Comment:
Is there a meaningful difference in the camel casing here e.g. *T*extCorpus vs. *t*reeReference?

Reply:
Yes, module and component names start with uppercase, property names start with lowercase.